bindings/python/.evergreen/test.sh

```diff
@@ -42,14 +50,23 @@ else
              "/opt/python/3.6/bin/python3"
              "/opt/python/pypy/bin/pypy"
              "/opt/python/pypy3.6/bin/pypy3")
+
+    ${PYTHONS[${#PYTHONS[@]}-3]} drivers-evergreen-tools/.evergreen/mongodl.py --component \
```

Comment:
`python3` does not work: 
```
[2022/05/21 02:03:18.610] + python3 drivers-evergreen-tools/.evergreen/mongodl.py --component csfle --version 6.0.0-rc4 --out ../csfle/ --target rhel70
[2022/05/21 02:03:18.611] Cloning into 'drivers-evergreen-tools'...
[2022/05/21 02:03:18.611] ./.evergreen/test.sh: line 56: python3: command not found
```
Testing `/opt/mongodbtoolchain/v3/bin/python3` now...

Reply:
`/opt/mongodbtoolchain/v3/bin/python3` works but runs into the error because of deterministic--I need to sed patch otherwise it does not work. 
```
[2022/05/21 02:49:00.117] `${SHELL_EXPANSION}` syntax will only work for Evergreen expansions.
[2022/05/21 02:49:00.120] + set -o errexit
[2022/05/21 02:49:00.120] + . .evergreen/utils.sh
[2022/05/21 02:49:00.120] + MONGOCRYPT_DIR=/data/mci/c9daba93ae0cbae83ef9433b0f6e4d11/all/rhel-70-64-bit
[2022/05/21 02:49:00.363] + git clone git@github.com:mongodb-labs/drivers-evergreen-tools.git
[2022/05/21 02:49:00.363] + '[' Windows_NT = '' ']'
[2022/05/21 02:49:00.364] ++ uname -s
[2022/05/21 02:49:00.364] + '[' Darwin = Linux ']'
[2022/05/21 02:49:00.364] + export PYMONGOCRYPT_LIB=/data/mci/c9daba93ae0cbae83ef9433b0f6e4d11/all/rhel-70-64-bit/nocrypto/lib64/libmongocrypt.so
[2022/05/21 02:49:00.364] + PYMONGOCRYPT_LIB=/data/mci/c9daba93ae0cbae83ef9433b0f6e4d11/all/rhel-70-64-bit/nocrypto/lib64/libmongocrypt.so
[2022/05/21 02:49:00.364] + PYTHONS=("/opt/python/2.7/bin/python" "/opt/python/3.4/bin/python3" "/opt/python/3.5/bin/python3" "/opt/python/3.6/bin/python3" "/opt/python/pypy/bin/pypy" "/opt/python/pypy3.6/bin/pypy3")
[2022/05/21 02:49:00.364] + export CSFLE_PATH=../csfle/lib/mongo_csfle_v1.so
[2022/05/21 02:49:00.364] + CSFLE_PATH=../csfle/lib/mongo_csfle_v1.so
[2022/05/21 02:49:00.364] + /opt/mongodbtoolchain/v3/bin/python3 drivers-evergreen-tools/.evergreen/mongodl.py --component csfle --version 6.0.0-rc4 --out ../csfle/ --target rhel70
[2022/05/21 02:49:01.509] Traceback (most recent call last):
[2022/05/21 02:49:01.509]   File "/data/mci/c9daba93ae0cbae83ef9433b0f6e4d11/libmongocrypt/bindings/python/drivers-evergreen-tools/.evergreen/mongodl.py", line 938, in <module>
[2022/05/21 02:49:01.510]     sys.exit(main(sys.argv[1:]))
[2022/05/21 02:49:01.510]   File "/data/mci/c9daba93ae0cbae83ef9433b0f6e4d11/libmongocrypt/bindings/python/drivers-evergreen-tools/.evergreen/mongodl.py", line 896, in main
[2022/05/21 02:49:01.510]     cache = Cache.open_in(args.cache_dir)
[2022/05/21 02:49:01.510]   File "/data/mci/c9daba93ae0cbae83ef9433b0f6e4d11/libmongocrypt/bindings/python/drivers-evergreen-tools/.evergreen/mongodl.py", line 465, in open_in
[2022/05/21 02:49:01.510]     db = CacheDB.open(dirpath / 'data.db')
[2022/05/21 02:49:01.510]   File "/data/mci/c9daba93ae0cbae83ef9433b0f6e4d11/libmongocrypt/bindings/python/drivers-evergreen-tools/.evergreen/mongodl.py", line 265, in open
[2022/05/21 02:49:01.510]     db.create_function('mdb_version_not_rc',
[2022/05/21 02:49:01.510] sqlite3.NotSupportedError: deterministic=True requires SQLite 3.8.3 or higher
```